### PR TITLE
Add StaticRoute CRD

### PR DESCRIPTION
**Pull request by TaoZou1**
Add StaticRoute CRD

### CODE REVIEW

File: api/v1alpha1/groupversion_info.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* Copyright © 2021 VMware, Inc. All Rights Reserved.
```
Comment (lxiaopei) [inline api/v1alpha1/groupversion_info.go:1]: There should be no changes for this file?

**Comment by lxiaopei** [comment]
Besides the _types.go file, we also need to generate manifests.

File: api/v1alpha1/staticroute_types.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	// Status defines status of condition type, True or False.
+	Status corev1.ConditionStatus `json:"status"`
+	// Reason shows a brief reason of condition.
+	Reason string `json:"reason,omitempty"`
+	// Message shows a human readable message about condition.
+	Message string `json:"message,omitempty"`
+}
+
+// StaticRouteSpec defines static routes configuration on Tier-0/Tier-1 or on VPC
+type StaticRouteSpec struct {
```
Comment (lxiaopei) [inline api/v1alpha1/staticroute_types.go:31]: we should define the CRD based on https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=NSBU&title=WCP+IaaS+Networking+v1+Design+Spec#WCPIaaSNetworkingv1DesignSpec-RouteTable

**Comment by TaoZou1** [comment]
Close this pull request since it not used code in clone repo. Create a new pull request https://github.com/vmware-tanzu/nsx-operator/pull/102

**Pull request closed without merging**